This failure is intermittent in this branch.

```
rerun 3
======================================================================= test process starts ========================================================================
executable:         /usr/bin/python  (2.7.6-final-0) [CPython]
architecture:       64-bit
cache:              yes
ground types:       python 
random seed:        83647877
hash randomization: on (PYTHONHASHSEED=3774641915)

sympy/solvers/tests/test_inequalities.py[1] F                                                                                                                 [FAIL]

____________________________________________________________________________________________________________________________________________________________________
______________________________________________ sympy/solvers/tests/test_inequalities.py:test_reduce_abs_inequalities _______________________________________________
  File "/home/debugger22/workspace/sympy/sympy/solvers/tests/test_inequalities.py", line 187, in test_reduce_abs_inequalities
    raises(TypeError, lambda: reduce_inequalities(abs(nr - 5) < 3))
  File "sympy/utilities/pytest.py", line 81, in raises
    raise AssertionError("DID NOT RAISE")
AssertionError: DID NOT RAISE
```

--- a/sympy/functions/elementary/complexes.py
+++ b/sympy/functions/elementary/complexes.py
@@ -495,10 +495,10 @@ class Abs(Function):
             return self.args[0].is_integer
 
     def _eval_is_nonzero(self):
-        return self._args[0].is_nonzero
+        return fuzzy_not(self._args[0].is_zero)
 
     def _eval_is_zero(self):
-            return self._args[0].is_zero
+        return self._args[0].is_zero
 
     def _eval_is_positive(self):
         is_z = self.is_zero

